src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/util.py

```diff
@@ -159,3 +178,88 @@ def guess_content_type(file_path, original, settings_class):
         content_language=original.content_language,
         content_md5=original.content_md5,
         cache_control=original.cache_control)
+
+
+def get_storage_client(cli_ctx, service_type, parsed_args):
+    from azure.cli.command_modules.storage._client_factory import get_storage_data_service_client
+
+    az_config = cli_ctx.config
+
+    name = getattr(parsed_args, 'account_name', az_config.get('storage', 'account', None))
+    key = getattr(parsed_args, 'account_key', az_config.get('storage', 'key', None))
+    connection_string = getattr(parsed_args, 'connection_string', az_config.get('storage', 'connection_string', None))
+    sas_token = getattr(parsed_args, 'sas_token', az_config.get('storage', 'sas_token', None))
+
+    return get_storage_data_service_client(cli_ctx, service_type, name, key, connection_string, sas_token)
+
+
+@Completer
+def file_path_completer(cmd, prefix, action, parsed_args, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Do these completers work?  The signature is supposed to be `def my_completer(cmd, prefix, namespace, [**kwargs])`

Reply:
What should be changed to the completer to make it conform the expected signature?

Reply:
Remove `action` and rename `parsed_args` to `namespace`.